frontend/src/views/Layout/ContentFooter.spec.js

```diff
@@ -59,6 +59,16 @@ describe('ContentFooter', () => {
           'https://github.com/gradido/gradido/releases/latest',
         )
       })
+
+      it('git commit hash and link', async () => {
```

Comment:
This seems a good start, just try to make a describe block for the it block and dive it in two it blocks. 
So the litteral would be:
mount:
- hash: 
-- has last commit hash
-- links to last release commit